js-plugins/sass/src/options.ts

```diff
@@ -2,7 +2,7 @@ import { isAbsolute, join } from 'path';
 import type { SassPluginOptions } from './index.js';
 import fs from 'fs';
 
-export const pluginName = 'farm-sass-plugin';
+export const pluginName = '@farmfe/js-plugin-sass';
```

Comment:
Does it feel better to take the name from package json if the name is the same now?

Reply:
Yes, I will modify it now